common/autoware_testing/design/autoware_testing-design.md

```diff
@@ -52,7 +52,7 @@ build/<package_name>/test_results/<package_name>/<executable_name>_smoke_test.xu
 
```

Comment:
Let me fix this typo as well. :bow: 
![image](https://user-images.githubusercontent.com/31987104/164413314-045f37ce-1448-44db-aa3c-5b337e861dc4.png)

Reply:
7480dadfeeb0c3a3015169e6f5651c46d801850b